### ISSUE TITLE
OperationalError: database is locked

### ISSUE THREAD

**Issue by Galtwise** [AHAAAAAAA/PokemonGo-Map#1028]
## Expected Behavior
Database won't lock with less than 10 concurrent instances

## Current Behavior
Instances go down one by one, with issue:
```
  cursor.execute(sql, params or ())
OperationalError: database is locked

```
## Possible Solution
Update SQL algorithm

## Steps to Reproduce (for bugs)

1. Run multiple instances
2. Wait
3. ????
4. Profit... err, failure

## Your Environment

* Environment name and version (e.g. Python 2.7): Python 2.7.6
* Operating System and version (desktop or mobile): Ubuntu 14.4

**Comment by vaurdan** [comment]
Duplicate of #744

**Comment by Galtwise** [comment]
You're right, jz6 asked me to duplicate it for him

**Comment by Galtwise** [comment]
Please don't mark as closed, as per JZ6

**Comment by rkowalk** [comment]
Galtwise - I'm unsure how the preferences are set for the sqlite3 DB, but it seems they are held in instantion.

I've altered the timeout as previous described, and then deleted the DB file(pogom.db).

Doing the above has given me ~3 hours of stability on 14 threads.

**Comment by MonkeyLeeT** [comment]
Fixed with #1596
